acra/acra-in-depth/data-structures/_index.md

```diff
@@ -2,3 +2,180 @@
 title: Data structures
 bookCollapseSection: true
 ---
+
+## AcraBlock
+
+### Container structure
```

Comment:
This should be moved a bit lower, after “To generate AcraBlocks Acra-Server uses symmetric keys”, where structure description actually starts.